js/scripts.js

```diff
@@ -1,2 +1,50 @@
-let hello = 'Привет';
-console.log(hello);
+const questionOne = {
+  question: 'Где обитает жираф?',
+  answer: {
+    a: 'В Индии',
+    b: 'В Америке',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/voronkova-ab/game-js/pullRequest?prid=7705697)
```suggestion
    b: "В Америке",
```